test/utils/metrics/metrics.go

```diff
@@ -131,6 +131,12 @@ func CheckClockRoleAndOffset(ptpConfig *ptpv1.PtpConfig, label, nodeName *string
 	if nodeName == nil {
 		var name string
 		name, err = getNode(*label)
+		if err != nil || name == "" || label == nil || *label != utils.PtpClockUnderTestNodeLabel {
```

Comment:
what would be the label incase of multi cluster for master(leader) and slave(follower) ? do you need to have that label defined